test/integration/targets/collections_runtime_pythonpath/runme.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -mvEeuxo pipefail
+
+
+export PIP_DISABLE_PIP_VERSION_CHECK=1
+TMP_DIR=`mktemp -d`
```

Comment:
Use the existing `OUTPUT_DIR` managed by `ansible-test` instead of creating a temp dir in the test.

Reply:
I need it outside of the path with a Unicode name because python 2 setuptools explode otherwise.

Reply:
What issue are you seeing? I didn't have any problems with the test using `OUTPUT_DIR`.

Reply:
Ah, it was failing for in-place builds. Seems to be okay with what you suggest.